project-base/.env

```diff
@@ -14,5 +14,23 @@
 # https://symfony.com/doc/current/best_practices/configuration.html#infrastructure-related-configuration
 
 IGNORE_DEFAULT_ADMIN_PASSWORD_CHECK='0'
-REDIS_PREFIX=''
+DATABASE_HOST=postgres
+DATABASE_PORT=5432
+DATABASE_NAME=shopsys
+DATABASE_USER=root
+DATABASE_PASSWORD=root
+ELASTICSEARCH_HOST='elasticsearch:9200'
 ELASTIC_SEARCH_INDEX_PREFIX=''
+REDIS_HOST=redis
+REDIS_PREFIX=''
+
+MAILER_TRANSPORT=smtp
+MAILER_HOST=smtp-server
+MAILER_USER=
+MAILER_PASSWORD=
+MAILER_DELIVERY_WHITELIST=/@shopsys\.com$/
+MAILER_DISABLE_DELIVERY=0
+MAILER_MASTER_EMAIL_ADDRESS=no-reply@shopsys.com
+
+SECRET=ThisTokenIsNotSoSecretChangeIt
```

Comment:
Symfony by default uses `APP_SECRET`. Is a reason to not follow with the same name?

Reply:
No reason. I kept original name. Anyway renaming it to `APP_SECRET` is really good point 👍

Reply:
fixed